### PR TITLE
Add ability to collect font information

### PR DESCRIPTION
**Pull request by jerbob92**
Add ability to collect font information

### CODE REVIEW

File: pdfium/internal/subprocess/text.go [hunk 1]
```diff
@@ -113,17 +126,32 @@ func (p *Pdfium) GetPageTextStructured(request *requests.GetPageTextStructured)
...
 			}
-			resp.Rects = append(resp.Rects, &responses.GetPageTextStructuredRect{
-				Text:          string(bytes.ReplaceAll(charData[0:charsWritten*4], []byte("\x00"), []byte{})),
-				PointPosition: pointPosition,
-				PixelPosition: convertPointPositions(request.PixelPositions, pointPosition, pointToPixelRatio),
-			})
+
+			if request.CollectFontInformation {
+				// Find index of the first letter of the rect.
+				// @todo: is 5 a "valid" tolerance?
```
Comment (Daan-Adrichem) [inline pdfium/internal/subprocess/text.go:141]: maybe add a issue where this is researched

File: pdfium/responses/responses.go [hunk 2]
```diff
@@ -33,17 +33,27 @@ type CharPosition struct {
 	Bottom float64
 }
 
+type FontInformation struct {
+	Size         float64 // Font size in points (also known as em)
+	SizeInPixels int     // Font size in pixels, only available when PixelPositions is used.
```
Comment (Daan-Adrichem) [inline pdfium/responses/responses.go:38]: maybe make this a pointer as it is not calculated and returning a 0 when we didn't calculate seems off.
eventhough 0 is an unrealistic value anyway

File: pdfium/internal/subprocess/text.go (hunk 1 above)
Comment (jerbob92) [reply pdfium/internal/subprocess/text.go:141]: Good one, added #23

File: pdfium/responses/responses.go (hunk 2 above)
Comment (jerbob92) [reply pdfium/responses/responses.go]: Yeah sounds better indeed, changed it :)

**Review by Daan-Adrichem — APPROVED** [review]

**Pull request merged**